src/providers/ghIssueProvider.ts

```diff
@@ -0,0 +1,36 @@
+import { displayWarning } from '#src/consoleUtils.js';
+import { execAsync } from '#src/utils.js';
+import type { ProviderConfig } from './types.js';
+
+/**
+ * Gets GitHub issue using GitHub CLI
```

Comment:
I think this, rather than jira should become our flagship feature in README.md. The implementation looks good. Now we need to update documentation, jira provider should only be mentioned in README.md and to be moved to docs/CONFIGURATION.md and this one should take jira's place in README.md